data/sql/updates/16_08_14_02.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE world_db_version CHANGE COLUMN 2016_08_14_00 2016_08_14_02 bit;
+
+UPDATE gameobject_template
+SET data10='29519' WHERE entry='181597'AND entry='181598'
```

Comment:
also you can't have a row having entry='181597' AND entry='181598' at the same time, probably you meant OR instead of AND...